src/dynarec/rv64/rv64_emitter.h

```diff
@@ -1029,11 +1029,29 @@ f28–31  ft8–11  FP temporaries                  Caller
 // rd2 := mem[addr+15:addr+8]
 #define TH_LDD(rd1, rd2, rs1, imm2) EMIT(R_type(0b1111100 | ((imm2)&0b11), rd2, rs1, 0b100, rd1, 0b0001011))
 
-// TODO
-// th.lwd rd1, rd2, (rs1), imm2, 3 Load two signed 32-bit values
-// th.lwud rd1, rd2, (rs1), imm2, 3 Load two unsigned 32-bit values
-// th.sdd rd1, rd2, (rs1), imm2, 4 Store two 64-bit values
-// th.swd rd1, rd2, (rs1), imm2, 3 Store two 32-bit values
+// Load two signed 32-bit values from memory into two GPRs.
+// addr := rs1 + (zero_extend(imm2) << 3)
+// reg[rd1] := sign_extend(mem[addr+3:addr])
+// reg[rd2] := sign_extend(mem[addr+7:addr+3])
```

Comment:
// reg[rd2] := sign_extend(mem[addr+7:addr+4])

Reply:
wow, this is interesting, I'm actually copied these pseudo codes directly from the thead spec.

Reply:
Fixed, I'll submit a PR to their spec repo.

Reply:
I still don't think it's `[addr+7:addr+3]` interval, that would be 36bits. Or mayb it's `[addr+7:addr+3[` but that's not ass clear as `[addr+7:addr+4]`

Reply:
Yeah, I think it's a typo, which is ugh... anyway...

Reply:
> Fixed, I'll submit a PR to their spec repo.

Done: https://github.com/T-head-Semi/thead-extension-spec/pull/29